src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -42,6 +48,18 @@ public static void initialize(Constants.Mode mode) {
                 kS.initDefault(0.0);
                 kV.initDefault(0.0);
                 kA.initDefault(0.0);
+                kG.initDefault(0.0);
```

Comment:
No need to initialize in sim if you don't use it

Reply:
e67b826